core/os/device/gpu_counter_descriptor.proto

```diff
@@ -37,10 +49,11 @@ message GpuCounterDescriptor {
     oneof peak_value {
       int64 int_peak_value = 5;
       double double_peak_value = 6;
-    };
+    }
```

Comment:
Is there a reason we delete the semicolon here?

Reply:
This is a copy of the file in Perfetto source code.  We shouldn't add a semicolon if it's not necessary.